src/main/java/io/stargate/sgv2/jsonapi/api/model/command/CommandContext.java

```diff
@@ -160,6 +167,18 @@ public ApiFeatures apiFeatures() {
     return apiFeatures;
   }
 
+  public FeatureUsage featureUsage() {
+    return featureUsage;
+  }
+
+  public void addFeature(Feature feature) {
+    this.featureUsage = this.featureUsage.withFeature(feature);
+  }
+
+  public void addFeatureUsage(FeatureUsage featureUsage) {
```

Comment:
see comments in the FeatureUsage class to make this mutable. 

As personal preference we the CommandContext should just probide access to the FeatureUsage, not provide helpers to call functions that are on it. That breaks encapsulation.